configure-freebsd.sh

```diff
@@ -0,0 +1,346 @@
+#!/usr/local/bin/bash
```

Comment:
I think we can simplify this script a lot for FreeBSD. Actually FreeBSD may be much closer to MacOS than to Linux, so maybe you can take `configure-mac_os_x.sh` script and modify it instead of doing all that work for the Linux configuration script

But please remember to replace "MacOS" with "FreeBSD"

Reply:
It is true. Mac OS X kernel is a hybrid of Mach and FreeBSD.

Reply:
Done.